src/sos/targets.py

```diff
@@ -136,11 +136,11 @@ def get(self, name, default=None):
     def __getattr__(self, name):
         try:
             return self._dict[name]
-        except Exception:
+        except Exception as name_in_self_dict:
```

Comment:
Please always use `as e`.

Reply:
I just checked the latest commit (212f2eb). It targets.py has 'e' instead of 'name_in_self_dict' on line 139. I apologize for the inconvenience.